crypto/blake2b/blake2x.go

```diff
@@ -13,11 +13,11 @@ import (
 // XOF defines the interface to hash functions that
 // support arbitrary-length output.
 type XOF interface {
-	// Write absorbs more data into the hash's state. It panics if called
+	// Writer absorbs more data into the hash's state. It panics if called
```

Comment:
The comment here referred to the method that the `io.Writer` interface brings to this composite interface, namely:
```golang
type Writer interface {
	Write(p []byte) (n int, err error)
}
```
So please revert this change and the one below re `io.Reader`